coffee-bom/pom.xml

```diff
@@ -103,6 +103,16 @@
 				<artifactId>weld-junit5</artifactId>
 				<version>${version.org.jboss.weld.weld-junit5}</version>
 				<scope>test</scope>
+				<exclusions>
+					<exclusion>
+						<groupId>javax.enterprise</groupId>
+						<artifactId>cdi-api</artifactId>
```

Comment:
cdi-api es javax.inject miert kellett bele? Ahogy neztem nincs is ilyen dependencyje es amugy is csak a test scope-ban van hasznalva

Reply:
jogos, amikor dolgoztam vele, még benne voltak (tranzitívan). azóta ahogy nézem volt egy verzió emelés amiben már alapból a jakarta párjuk található. kiszedem.